ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,103 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private-postupg:
```

Comment:
@mgahagan73 I guess your file name is not correct. it should have `nightly-4.14-upgrade-from-<build type>-<release number>`
if I am wrong, please @liangxia correct me.

Reply:
The job in prow that provisions ARO currently installs the default version set by the ARO SRE team.  I think they currently default to installing 4.10.54 so we are upgrading in steps to get to the latest nightly (once 4.13 is officially released I'll switch 4.13 to the stable channel).  The file name is based on the existing 4.11/12/13 aro upgrade jobs.

ARO team recently released a feature to allow the desired version (selected from versions supported by SRE's in a given region) to be specified on the az command line. I have not yet upgraded the aro provisoiner in Prow to allow for version selection, that will be the subject of a future PR as it depends on a new version of az-cli than we currently have in Prow.

Reply:
ok, got it. thanks